### PR TITLE
Updated README.md with independent flag

### PR DESCRIPTION
**Pull request by blumk**
I've updated JSON example only. Once dist files are updated, I'll update the other examples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ This repository contains lists of world countries in JSON, CSV and XML. Each lin
  - code ISO 3166-1 numeric (`ccn3`)
  - code ISO 3166-1 alpha-3 (`cca3`)
  - code International Olympic Committee (`cioc`)
+ - Independent or not in ISO 3166 (`independent`)
```
Comment (mledoze) [inline README.md]: @blumk could you please change the text to: 

independence status from ISO 3166-1 (`independent`) (denotes the country is considered a [sovereign state](https://en.wikipedia.org/wiki/Sovereign_state)")

Comment (blumk) [reply README.md]: @mledoze Fixed. Thanks!

**Comment by blumk** [comment]
@mledoze I've added PR #267 (status property) containing changes from this PR. If you decide to add PR #267 then you can abandon this PR.

**Comment by mledoze** [comment]
I close this PR, we'll work on #267 instead.

**Pull request closed without merging**